src/utility/json_export.rb

```diff
@@ -22,7 +22,7 @@ def self.graph_to_json(triangle = nil, animation, spring_constants_for_ids, moun
     end
     json[:spring_constants] = spring_constants_for_ids if spring_constants_for_ids
     json[:standard_surface] = triangle.nodes_ids_towards_user
-    json[:mounted_users] = mounted_users if mounted_users
+    json[:mounted_users] = mounted_users_to_hash(Graph.instance.nodes)
```

Comment:
I really like that we just use :mounted_users everywhere now, for saving as well as for parsing in the server